Std/Util/LibraryNote.lean

```diff
@@ -26,8 +26,9 @@ initialize libraryNoteExt : SimplePersistentEnvExtension LibraryNoteEntry (Array
 open Lean Parser Command in
 /--
 ```
-/-- ... some explanation ... -/
-library_note "some tag"
+library_note "some tag"/--
```

Comment:
```suggestion
library_note "some tag" /--
```
I think the documentation is my fault.. I just copied it from mathlib3 when I ported `library_note`.